plugin.js

```diff
@@ -25,7 +25,7 @@ CKEDITOR.plugins.add( 'bylawlist', {
         editor.ui.addButton( 'Bylawlist', {
             label: 'Add Bylawlist',
             command: 'insertBylawlist',
-            toolbar: 'insert'
+            toolbar: 'insert'g
```

Comment:
Is that g supposed to be there?

Reply:
looks like a typo...did the pull request just before I shutdown on Friday, may have been a bit in a hurry